### PR TITLE
moved to new version of combine, OS 9

### PR DESCRIPTION
**Pull request by ahmad3213**

### CODE REVIEW

File: env.sh [hunk 1]
```diff
@@ -170,8 +171,8 @@ action() {
   local os_prefix=$(get_os_prefix $os_version)
   local node_os=$os_prefix$os_version
 
-  # local default_cmssw_ver=CMSSW_13_0_15
-  local default_cmssw_ver=CMSSW_14_0_8
+  local default_cmssw_ver=CMSSW_13_0_15
```
Comment (kandrosov) [inline env.sh:174]: why do you roll back to 13_0_15?

File: env.sh [hunk 2]
```diff
@@ -99,7 +99,8 @@ EOF
   run_cmd cd "$cmssw_base/src"
   run_cmd eval `scramv1 runtime -sh`
   if ! [ -d "CombineHarvester" ]; then
-    run_cmd bash <(curl -s https://raw.githubusercontent.com/cms-analysis/CombineHarvester/main/CombineTools/scripts/sparse-checkout-ssh.sh)
+    #run_cmd bash <(curl -s https://raw.githubusercontent.com/cms-analysis/CombineHarvester/main/CombineTools/scripts/sparse-checkout-ssh.sh)
+    run_cmd bash <(https://raw.githubusercontent.com/nucleosynthesis/work-tools/master/sparse-checkout-SL-ssh.sh)
```
Comment (kandrosov) [inline env.sh:103]: - please use CombineHarvester from the central repository https://github.com/cms-analysis/CombineHarvester
- It looks like for the newest release, a sparse checkout is not needed, and one can directly use 
  ```
  git clone https://github.com/cms-analysis/CombineHarvester.git CombineHarvester
  ```

File: env.sh [hunk 3]
```diff
@@ -180,26 +181,23 @@ action() {
...
   fi
-
   source /afs/cern.ch/work/k/kandroso/public/flaf_env/bin/activate
   source /cvmfs/sft.cern.ch/lcg/app/releases/ROOT/6.30.06/x86_64-almalinux9.3-gcc114-opt/bin/thisroot.sh
   run_cmd source /afs/cern.ch/user/m/mrieger/public/law_sw/setup.sh
   source "$( law completion )"
   source /cvmfs/cms.cern.ch/rucio/setup-py3.sh &> /dev/null
-
   alias cmsEnv="env -i HOME=$HOME ANALYSIS_PATH=$ANALYSIS_PATH ANALYSIS_DATA_PATH=$ANALYSIS_DATA_PATH X509_USER_PROXY=$X509_USER_PROXY CENTRAL_STORAGE=$CENTRAL_STORAGE ANALYSIS_BIG_DATA_PATH=$ANALYSIS_BIG_DATA_PATH DEFAULT_CMSSW_BASE=$DEFAULT_CMSSW_BASE DEFAULT_CMSSW_ARCH=$DEFAULT_CMSSW_ARCH $ANALYSIS_PATH/RunKit/cmsEnv.sh"
   alias cmbEnv="env -i HOME=$HOME ANALYSIS_PATH=$ANALYSIS_PATH ANALYSIS_DATA_PATH=$ANALYSIS_DATA_PATH X509_USER_PROXY=$X509_USER_PROXY CENTRAL_STORAGE=$CENTRAL_STORAGE ANALYSIS_BIG_DATA_PATH=$ANALYSIS_BIG_DATA_PATH /cvmfs/cms.cern.ch/common/cmssw-cc7 -- $ANALYSIS_PATH/cmb_env.sh"
```
Comment (kandrosov) [inline env.sh:202]: singularity is no longer needed to run inference

**Review by kandrosov — CHANGES_REQUESTED** [review]

**Comment by ahmad3213** [comment]
i mistakenly was pushing new commits to this one. i will open a new PR to avoid confusion taking care of your comments

**Pull request closed without merging**